apps/api/v1/pages/api/slots/_get.test.ts

```diff
@@ -66,11 +66,8 @@ describe("GET /api/slots", () => {
         buildMockData();
         await handler(req, res);
         console.log({ statusCode: res._getStatusCode(), data: JSON.parse(res._getData()) });
-        expect(JSON.parse(res._getData())).toMatchInlineSnapshot(`
-          {
-            "slots": {},
```

Comment:
The intentiona seems just to be verifying `slots` prop to be there but now slots property can't be empty if the user is there.

Reply:
cc @zomars

Reply:
As long as there's always a slot properly even if there's no available ones were good. For preventing breaking changes @hariombalhara